vite.config.ts

```diff
@@ -11,6 +11,7 @@ export default defineConfig({
     preprocessorOptions: {
       scss: {
         includePaths: [path.resolve('./src/sass/')],
+        pkgImporter: new NodePackageImporter('.'),
```

Comment:
For some reason, without this `'.'` I was getting the error:

```
failed to load config from /Users/jonny/Sites/oddcontrast/vite.config.ts
error during build:
The Node package importer cannot determine an entry point because `require.main.filename` is not defined. Please provide an `entryPointDirectory` to the `NodePackageImporter`.
```

Reply:
Interesting, yeah that makes sense 👍